client/src/components/Header/Header.js

```diff
@@ -1,52 +1,11 @@
 import React from "react";
```

Comment:
You don't need to import react anymore in the newest react version so this line of code is obsolete.

Reply:
Sometimes it works, sometime it doesn´t.. 😄